cli/tests/unit/files_test.ts

```diff
@@ -263,38 +263,6 @@ Deno.test(
   },
 );
 
-Deno.test(
-  { permissions: { write: true } },
-  async function writeSyncWhileAsyncFails() {
-    const tempDir = await Deno.makeTempDir();
-    try {
-      const filePath = tempDir + "/file.txt";
-      const file = await Deno.open(filePath, { create: true, write: true });
-      const rid = file.rid;
-      try {
-        // set a file lock so the async write will be held up
-        await Deno.flock(rid, true);
```

Comment:
I wasn't thinking when I wrote this.

The test below would only pass when the Deno.write happened to hold the resource while the Deno.writeSync occurred. If it took too long or too short then the test would fail.